modules/security-aai/pom.xml

```
@@ -59,10 +70,12 @@
   </dependencies>
   <build>
     <plugins>
+      <!--
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-dependency-plugin</artifactId>
       </plugin>
+      -->
```

Add exceptions if necessary but don't just uncomment this
```suggestion
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-dependency-plugin</artifactId>
      </plugin>
```

---

Exceptions were added.